src/content/docs/guides/configure-biome.mdx

```diff
@@ -187,33 +178,36 @@ Some shells don't support the recursive glob `**`.
 :::
 
 The Biome configuration file can be used to refine which files are processed.
-You can explicitly list the files to be processed using `include` and the files not to be processed using `ignore`.
-`include` and `ignore` accepts globs patterns such as `src/**/*.js`.
-See the [related section](#glob-syntax-explained) for which glob syntaxes are supported.
-`include` is always applied first before applying `ignore`.
-This allows you to include some files and to ignore some of the file you included.
-
-:::note
-`include` and `ignore` have a slightly different semantics.
-`include` doesn't prevent Biome of traversing a folder.
-This means that if you want to prevent Biome from traversing a folder, you have to add the folder to `ignore`.
+You can explicitly list the files to be processed using
+[the `files.includes` field](/reference/configuration). `files.includes` accepts
+[glob patterns](/reference/configuration#glob-syntax-reference) such as
+`src/**/*.js`. Negated patterns starting with `!` can be used to exclude files.
+
+Paths and globs inside Biome's configuration file are always resolved relative
+to the folder the configuration file is in.
+
+`files.includes` applies to all of Biome's tools, meaning the files specified
+here are processed by the linter, the formatter and the assist, unless specified
+otherwise. For the individual tools, you can further refine the matching files
+using `<tool>.includes`.
+
+:::important
+The global `files.includes` has slightly different semantics than the more
+specific `<tool>.includes` fields. Any file or folder that doesn't match
+`files.includes` is excluded from use by any of Biome's tools. This means that
+any tool-specific `includes` field can never match a file that doesn't also
+match `files.includes`.
```

Comment:
It seems that this paragraph repeats what was stated before, just in different words. 

> `files.includes` applies to all of Biome's tools, meaning the files specified
here are processed by the linter, the formatter and the assist, unless specified
otherwise. For the individual tools, you can further refine the matching files
using `<tool>.includes`.

To me this phrase is clear enough, so maybe this paragraph should talk about `<tool>.includes` instead?

Reply:
It was primarily meant to clarify what “refinement” means, since I felt it was far from obvious. The fact that tool-specific files cannot match things outside the global files bears some explicit calling out, I think.